internal/broker/broker.go

```diff
@@ -70,12 +70,15 @@ type session struct {
 
 	oidcServer   *oidc.Provider
 	oauth2Config oauth2.Config
-	authInfo     map[string]any
 	isOffline    bool
 	userDataDir  string
 	passwordPath string
 	tokenPath    string
 
+	// Data to pass from one request to another.
+	deviceAuthResponse *oauth2.DeviceAuthResponse
+	authInfo           *token.AuthCachedInfo
+
```

Comment:
But I can understand why these changes make the code more readable and workable, so I agree with them.